.gitignore

```diff
@@ -1,2 +1,4 @@
 debug.test
 vendor
+
+.idea/
```

Comment:
Hello!

This is not the right place to place this.  Can you have a look at this https://gist.github.com/subfuzion/db7f57fff2fb6998a16c ? And please remove this commit. Thanks